zephyr/src/supp_main.c

```diff
@@ -13,19 +13,31 @@
  */
 
 #include <logging/log.h>
+#include <sys/fcntl.h>
 LOG_MODULE_REGISTER(wpa_supplicant, LOG_LEVEL_DBG);
 
 #include "includes.h"
 #include "common.h"
+#include "eloop.h"
 #include "wpa_supplicant/config.h"
 #include "wpa_supplicant_i.h"
-#include "driver_i.h"
+
 #include "fst/fst.h"
 #include "includes.h"
 #include "p2p_supplicant.h"
 #include "wpa_supplicant_i.h"
+#include "driver_i.h"
+
+#include "supp_main.h"
+
+
+#define DUMMY_SOCKET_PORT 9999
+
+#define DUMMY_SOCKET_PORT 9999
```

Comment:
This seems to be defined twice.

Reply:
Urgh..fixed this in another branch, too many open branches :).